.drone.yml

```diff
@@ -51,6 +51,37 @@ steps:
   environment:
     CGO_ENABLED: 0
 
+- name: release-snapshot-dev
```

Comment:
For the reviewer: These newly added steps ensure `dev` tagged container up-to-date. That particular container is used to test the plugin on its on CI pipeline.